README.md

```diff
@@ -19,267 +19,173 @@ REQUIRED SECTIONS:
 - Acknowledgements
 - License
 
-After you're finished please remove all the comments and instructions!
--->
-
-<div align="center">
-
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
   - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# CoinStatistics  <a name="about-project"></a>
 
-**[your_project__name]** is a...
+> It is a web app which show top crypto coins stats live. 
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+>HTML
+>Javascript
+>CSS
+>Webpack
+>React
+>Tailwind
+>Jest
+>React Router
+>Redux
+>React Testing Library
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- Actions and state management using React Redux
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- API integration using the [https://api.coinstats.app/public/v1/coins]
+
+- Homepage listing all Crypto currencies and a details page displaying more Crypto info.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+> Link to the deployed project 👇
+
+- 👉 Live Demo of Website :- [@Live_Demo_Render](https://coin-statistics.onrender.com)
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- 👉 Loom Video about this Project :- [@Loom_Video_Link](https://drive.google.com/file/d/1Xz6Q_GQCQ2lteLjA4Vgfrm_DDV31nW54/view?usp=sharing)
```

Comment:
- [ ] You did a great job so far @harshitajain06, But as the previous code reviewer mentioned, your video lacks a piece of code that's interesting or makes you proud. According to the main requirements. Please understand this video is a part of your project documentation and needs to give an over all overview.